### PR TITLE
Fixes issue with bias charts auto-refreshing with stale data (#1403)

### PR DESCRIPTION
**Pull request by alexcreasy**
* Closes: #1403 

## Description

Fixes bug described in ticket #1403 -- please read for detailed description

## How Has This Been Tested?

* Go to the bias metric charts page, select a chart to display
* Set the timeframe to 1 hour and refresh interval to 1 minute, to make testing easier.
* Leave the chart open for a few minutes, check there is no gap in the data as seen in the image below.

## Test Impact

No tests for this feature yet, so none to change.

## Request review criteria:

- [x] The commits have meaningful messages (squashes happen on merge by the bot).
- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work
- [ ] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

### CODE REVIEW

File: frontend/src/utilities/useRefreshInterval.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const useRefreshInterval = (refreshInterval, callback: () => void) => {
```
Comment (DaoDaoNoCode) [inline frontend/src/utilities/useRefreshInterval.ts:3]: `refreshInterval` type should be `number`?

**Review by DaoDaoNoCode — CHANGES_REQUESTED** [review]
Some small suggestions! Otherwise it looks good

File: frontend/src/utilities/useRefreshInterval.ts (hunk 1 above)
Comment (alexcreasy) [reply frontend/src/utilities/useRefreshInterval.ts:3]: That gives a type error, it seems to think it's a NodeJS.Timer which doesn't return a number - this was how I got round it. I wonder if there's a better way of doing it to force it to use the browser timer instance, maybe calling window.setInterval? WDYT?

**Review by DaoDaoNoCode — APPROVED** [review]
lgtm

**Pull request merged**